lib/src/rules/avoid_renaming_method_parameters.dart

```diff
@@ -55,18 +55,29 @@ class AvoidRenamingMethodParameters extends LintRule implements NodeLintRule {
   @override
   void registerNodeProcessors(
       NodeLintRegistry registry, LinterContext context) {
-    final visitor = _Visitor(this);
+    final visitor = _Visitor(this, context);
     registry.addMethodDeclaration(this, visitor);
+    registry.addCompilationUnit(this, visitor);
   }
 }
 
 class _Visitor extends SimpleAstVisitor<void> {
   final LintRule rule;
+  final LinterContext context;
 
-  _Visitor(this.rule);
+  bool isInLib;
```

Comment:
I think this warrants a comment; just saying that it specifically states whether the _compilation unit_ is in [context]'s package.

Reply:
Sure thing.  Will do.